projects/components/src/table/table.component.ts

```diff
@@ -149,13 +151,16 @@ export class TableComponent
   implements
     OnChanges,
     AfterViewInit,
+    OnDestroy,
     ColumnConfigProvider,
     TableDataSourceProvider,
     FilterProvider,
     ColumnStateChangeProvider,
     RowStateChangeProvider {
   private static readonly PAGE_INDEX_URL_PARAM: string = 'page';
   private static readonly PAGE_SIZE_URL_PARAM: string = 'page-size';
+  private static readonly SORT_COLUMN_URL_PARAM: string = 'sort';
```

Comment:
sort-attribute? or sort-on ?

Reply:
updated to `sort-by` based on offline conversation